code/src/CoreTemplateStudio/CoreTemplateStudio.Core/Diagnostics/WizardTypeEnum.cs

```diff
@@ -6,10 +6,10 @@ namespace Microsoft.Templates.Core.Diagnostics
 {
     public enum WizardTypeEnum
     {
-        NewProject = 0,
-        AddPage = 1,
-        AddFeature = 2,
-        AddService = 3,
-        AddTesting = 4,
+        NewProject,
+        AddPage,
+        AddFeature,
+        AddService,
+        AddTesting,
```

Comment:
Best review and change this on a separate issue
```suggestion
NewProject = 0,
AddPage = 1,
AddFeature = 2,
AddService = 3,
AddTesting = 4,
```